### DISCUSSION TITLE
Mention the sources in the media attribution statements

### DISCUSSION THREAD

**Discussion by obulat** [WordPress/openverse#3371] [General]
In the attribution statement, we share the parts that are described in the CC guidelines[^1]: Title, Author (with a link if available), Source (the media's landing URL) and License.  While this is sufficient, I believe we should expand the attribution statement to also include the provider of the media. This is described in the CC guidelines for attributing the public domain materials [^2]

My suggestion is to update the attribution from the current version
"[The Milkmaid](https://hdl.handle.net/10934/RM0001.COLLECT.6417)" by [Johannes Vermeer](https://www.rijksmuseum.nl/en/search?p=1&ps=12&involvedMaker=Johannes%20Vermeer&st=Objects) is marked with [Public Domain Mark 1.0](https://creativecommons.org/publicdomain/mark/1.0/?ref=openverse).

to a more complete one that includes the provider who digitized this work:

"[The Milkmaid](https://hdl.handle.net/10934/RM0001.COLLECT.6417)" by [Johannes Vermeer](https://www.rijksmuseum.nl/en/search?p=1&ps=12&involvedMaker=Johannes%20Vermeer&st=Objects). The Rijksmuseum collection. [Public Domain Mark 1.0](https://creativecommons.org/publicdomain/mark/1.0/?ref=openverse).

What do you think, @WordPress/openverse-maintainers ?

[^1]: [Best practices for attribution CC wiki page](https://wiki.creativecommons.org/wiki/best_practices_for_attribution)
[^2]: [Attributing the public domain materials](https://wiki.creativecommons.org/wiki/best_practices_for_attribution#Attributing_public_domain_materials)

**Comment by sarayourfriend** [comment]
Strong "yes!" from me. This would be an excellent change to go hand-in-hand with improvements to Wikimedia Commons "source" identification to get better information there as well. So much stuff is PDM in Wikimedia Commons but from amazing sources like National Art Gallery (of the US) with deep collections and hardworking teams of digital archivists that don't get credited even by under their institutional name on Openverse.

**Reply by fcoveram** [reply]
Algo yes from my side. But I would include the `is marked with` part as the other license's variants have `is licensed under`. The [Attributing public domain materials section](https://wiki.creativecommons.org/wiki/best_practices_for_attribution#Attributing_public_domain_materials) in the best practices article has an example of a work from Rijksmuseum. The wording is different but points out how the license relates to it.

**Reply by sarayourfriend** [reply]
Is it time to create an issue to implement this @obulat?
